earth_enterprise/rpms/shared/lib/read-xml.sh

```diff
@@ -0,0 +1,340 @@
+# Copyright 2017 the Open GEE Contributors.
```

Comment:
@tst-ppenev can we look for a solution that doesn't involve parsing XML in bash?  Surely there's a tool that can do it for us that's available on all platforms.  I just worry that writing our own XML parsing library will lead to trouble down the road.

Reply:
We can have a spike to evaluate XML tools, if you want.

What can of trouble are you expecting? I'm not sure it makes that much of a difference what an XML parser is implemented in.

Reply:
@tst-ppenev  From my perspective, I simply want to make sure what we choose to do is both maintainable and consistent on all the platforms it is needed on.  I am not particularly concerned what it is written in, but if there is something we can already use, then that is something we do not have to also actively maintain.

Reply:
Well, it doesn't make much of a difference to me.

I know this would work to finish the install scripts, and it has a minimal amount of dependencies.  I don't know what the best tool we can use on the needed platforms is.

I think we may have enough to finish the packaging tasks.  We're only using basic features, which I don't expect to change in the XML standard soon.  So, we may be able to collect all the concerns to address, and add a follow-up task.

Reply:
My proposal is to use this function. It calls xmllint, which is available on all our target platforms, and should be compatible with the older version of xmllint installed on rhel 6 and centos 6. I tested on Ubuntu 16.04 and RHEL6, so hopefully that is a representative sample. 
```
#!/bin/bash

get_xpath()
{
  local FILE="$1"
  local XPATH="$2"
  
  local OUTPUT=$(echo "cat $XPATH" | xmllint --shell "$FILE" | sed '/^\/ >/d' | sed 's/<[^>]*.//g')
  echo $OUTPUT 
}

echo $(get_xpath '/opt/google/etc/systemrc' '//Systemrc/assetroot/text()')
```

Reply:
This is most consistent with what we had used before...

Reply:
I like it!

Reply:
I already used  get_xpath in my pull request (#585), so we can either merge my branch into this, or approve this and then rebase and take mine out of wip...